src/test/java/com/epam/tc/hw3/tests/ex2/DifferentElementsPageTest.java

```diff
@@ -0,0 +1,54 @@
+package com.epam.tc.hw3.tests.ex2;
+
+import com.epam.tc.hw3.pages.DifferentElementsPage;
+import com.epam.tc.hw3.pages.HomePage;
+import com.epam.tc.hw3.tests.utils.BaseTest;
+import com.epam.tc.hw3.tests.utils.TestProperties;
+import org.testng.annotations.Test;
+
+public class DifferentElementsPageTest extends BaseTest {
+
+    @Test
+    public void differentElementsPageTest() {
+        HomePage homePage = new HomePage(webDriver);
+
+        // 1. Open test site by URL
+        assertHomePageURL();
+
+        // 2. Assert Browser title
+        assertHomePageTitle();
+
+        // 3, 4. Perform login and assert Username is loggined
+        homePage.getHeader().login();
+        softAssert.assertEquals(homePage.getHeader().getUserName(),
+            TestProperties.getTestProperties().getProperty("fullName"));
```

Comment:
First 4 verifications are the same in both tests. The should me moved to common before.